tests/unit/virtualrouter/test_vr_plugin_handlers.py

```
@@ -0,0 +1,640 @@
+        plugin = dnsmasq_module.Dnsmasq.__new__(dnsmasq_module.Dnsmasq)
+        plugin.signal_count = 0
+        plugin.config = MagicMock()
+        # Make restartDnsmasqAfterNumberOfSIGUSER1 > signal_count
+        plugin.config.init_command.restartDnsmasqAfterNumberOfSIGUSER1 = 100
+
+        linux = sys.modules["zstacklib.utils.linux"]
+        linux.get_pid_by_process_name.return_value = "1234"
+        linux.is_systemd_enabled.return_value = True
+        linux.sync_file.return_value = None
+
+        shell = sys.modules["zstacklib.utils.shell"]
```

Ruff (0.15.2)

[error] Local variable `shell` is assigned to but never used

Remove assignment to unused variable `shell`

(F841)